flock_internal_test.go

```diff
@@ -0,0 +1,30 @@
+package flock
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+func Test(t *testing.T) {
+	tmpFileFh, err := ioutil.TempFile(os.TempDir(), "go-flock-")
+	tmpFileFh.Close()
+	tmpFile := tmpFileFh.Name()
+	os.Remove(tmpFile)
+
+	lock := New(tmpFile)
+	locked, err := lock.TryLock()
+	if locked == false || err != nil {
+		t.Fatal("failed to lock")
+	}
+
+	newLock := New(tmpFile)
+	locked, err = newLock.TryLock()
+	if locked != false || err != nil {
+		t.Fatal("should have failed locking")
```

Comment:
Similar to my feeling about the above check, what do you think about `t.Fatalf("should have failed locking: locked: %t, err: %v", locked, err)`

Reply:
```suggestion
		t.Fatalf("should have failed locking: locked: %t, err: %v", locked, err)
```